apps/aehttp/src/aehttp_helpers.erl

```diff
@@ -525,28 +523,6 @@ get_block_from_chain(Fun) when is_function(Fun, 0) ->
             {404, [], #{reason => <<"Not mining, no pending block">>}}
     end.
 
-get_block(Fun, Req) ->
-    get_block(Fun, Req, true).
-
-get_block(Fun, _Req, AddHash) when is_function(Fun, 0) ->
-    case get_block_from_chain(Fun) of
-        {ok, Block} ->
-            Resp0 = aehttp_logic:cleanup_genesis(
```

Comment:
aehttp_logic:cleanup_genesis is now unused and shall be deleted?

Are there tests on whether and how the following field are serialized for the client for the genesis block for the various APIs?
- prev_hash
- miner
- beneficiary
- pow
- nonce

Reply:
Could you please add ticket (post-release) tracking adding tests re serialization of genesis in user API? So that we can confidently document this if we want to (e.g. as comment in swagger.yaml).

Reply:
PT: https://www.pivotaltracker.com/story/show/160148655